xml/operations-maintenance-upgrade_maintenance.xml

```diff
@@ -0,0 +1,1828 @@
+
+ardana-cp-mml-m1****************************************************************
+SUCCESS: Keystone V2 ==> V3 API config changes detected.
+ardana-cp-mml-m2****************************************************************
+SUCCESS: Keystone V2 ==> V3 API config changes detected.
+ardana-cp-mml-m3****************************************************************
+SUCCESS: Keystone V2 ==> V3 API config changes detected.
+localhost***********************************************************************
+     </screen>
+     <para>
+       The report states the following:
+     </para>
+     <variablelist>
+       <varlistentry>
+         <term>SUCCESS: Keystone V2 ==> V3 API config changes detected.</term>
+         <listitem>
+           <para>
+             This check confirms that your cloud has been updated with the
+             necessary changes such that all services will be using Keystone V3 API.
+             This means that there should be minimal interruption of service during
+             the upgrade. This is important because the Keystone V2 API has been
+             removed in &productname; &productnumber;.
+           </para>
+         </listitem>
+       </varlistentry>
+       <varlistentry>
+         <term>NOTE: pre-upgrade-swift-checks: Swift XFS inode size of 256 for
+           /srv/node/disk0 is smaller than SLE 12 SP4 recommended 512. Some
+           recommended XFS data integrity features may not be available after upgrade.</term>
+         <listitem>
+           <para>
+             This check will only report something if you have local &o_objstore;
+             configured and it is formatted with the SLE 12 SP3 default XFS
+             inode size of 256. In SLE 12 SP4, the default XFS inode size for a
+             newly formatted XFS file system has been increased to 512, to
+             allow room for enabling some additional XFS data-integrity features by default.
+           </para>
+         </listitem>
+       </varlistentry>
+     </variablelist>
+   </step>
+ </procedure>
+ <note>
+   <para>
+     There will be no loss of functionality as regards the &o_objstore;
+     solution after the upgrade. The difference is that some additional
+     XFS features will not be available on file systems which were formatted
+     under SLE 12 SP3 or earlier. These XFS features aid in the detection of,
+     and recovery from, data corruption. They are enabled by default for XFS
+     file systems formatted under SLE 12 SP 4.
+   </para>
+ </note>
+ <procedure>
+   <title>Additional Pre-Upgrade Checks That Should Be Performed</title>
+   <para>
+     In addition to the automated upgrade checks above, there are some checks
+     that should be performed manually.
+   </para>
+   <step>
+     <para>
+       For each network interface device specified in the input model under
+       <filename>~ardana/openstack/my_cloud/definition</filename>, ensure that
```

Comment:
This path looks incorrect to me.

`~` means the home directory, so surely it should be something like:

`~/ardana/openstack/my_cloud/definition`

or if it is a hidden folder,

`~/.ardana/openstack/my_cloud/definition`

...?

Reply:
For clarity: AIUI, `~ardana` would resolve to something like:

`/home/lprovenardana`

Which is almost certainly invalid.